commcare_connect/opportunity/visit_import.py

```diff
@@ -125,7 +125,7 @@ def get_status_by_visit_id(dataset) -> dict[int, VisitValidationStatus]:
     for row in dataset:
         row = list(row)
         visit_id = str(row[visit_col_index])
-        status_raw = row[status_col_index].lower()
+        status_raw = row[status_col_index].lower().replace(" ", "_")
```

Comment:
I was also thinking of adding a trim here to avoid converting unnecessary spaces to underscore